doc/nrf/ug_bt_mesh.rst

```diff
@@ -0,0 +1,91 @@
+The support is based on Zephyr's :ref:`bluetooth_mesh` implementation.
+
+.. _bt_mesh_ug_intro:
+
+Introduction
+************
+
+The `Bluetooth Mesh Profile Specification`_ is developed and published by the Bluetooth SIG.
+It allows one-to-one, one-to-many, and many-to-many communication, using
+the Bluetooth Low Energy protocol to exchange messages between the nodes on the network.
+All nodes in a Bluetooth Mesh network can communicate with each other, as long as there's a chain of nodes between them to relay the messages.
+The messages are encrypted with two layers of 128 bit AES-CCM encryption, allowing secure communication between thousands of devices.
+
+The end-user applications are implemented as a set of Mesh Models.
+The Bluetooth SIG defines some generic and reusable models in the `Mesh Model Specification`_, but vendors are free to define their own models.
+
+Read more about the Bluetooth Mesh specification on the Bluetooth SIG website:
+
+* `Bluetooth Mesh Glossary`_
+* `Bluetooth Mesh FAQ`_
+* `Bluetooth Mesh Study Guide`_
+
+.. _mesh_ug_supported features:
+
+Supported features
+******************
+
+The Bluetooth Mesh in |NCS| supports all mandatory features of the `Bluetooth Mesh Profile Specification`_ and the `Mesh Model Specification`_, as well as most of the optional features.
+
+The following features are supported by Zephyr's :ref:`zephyr:bluetooth_mesh`:
+
+* :ref:`zephyr:bluetooth_mesh_core`:
+
+  * Node role
+  * Relay feature
+  * Low power feature
+  * Friend feature
+
+* :ref:`zephyr:bluetooth_mesh_provisioning`:
+
+  * Provisionee role (GATT and Advertising Provisioning bearer)
+  * Provisioner role (only Advertising Provisioning bearer)
+
+* :ref:`GATT Proxy Server role <zephyr:bt_mesh_proxy>`
+* :ref:`zephyr:bluetooth_mesh_models`
+
+In addition to these features, |NCS| implements several models defined in the  `Mesh Model Specification`_, see :ref:`bt_mesh_models` for details.
+
+Configuration
+*************
+
+The Bluetooth Mesh support is controlled by :option:`CONFIG_BT_MESH`, which depends on the following configuration options:
+
+* :option:`CONFIG_BT`
+* :option:`CONFIG_BT_OBSERVER`
+* :option:`CONFIG_BT_PERIPHERAL`
+
+The Bluetooth Mesh subsystem currently performs best with :ref:`Zephyr's Bluetooth LE Controller <zephyr:bluetooth-arch>`.
+To force |NCS| to build with this controller, enable :option:`CONFIG_BT_LL_SW_SPLIT`.
+
+
+Optional configuration options
+==============================
+
+Optional features in the Bluetooth Mesh stack must be explicitly enabled:
+
+* :option:`CONFIG_BT_MESH_RELAY`
+* :option:`CONFIG_BT_MESH_FRIEND`
+* :option:`CONFIG_BT_MESH_LOW_POWER`
+* :option:`CONFIG_BT_MESH_PROVISIONER`
+* :option:`CONFIG_BT_MESH_GATT_PROXY`
+* :option:`CONFIG_BT_MESH_PB_GATT`
+* :option:`CONFIG_BT_MESH_CDB`
+
+The |NCS| Bluetooth Mesh Model implementations are also optional features, and must be explicitly enabled to be included in the build.
```

Comment:
What does this mean, practically?
Isn't there a contradiction between this "are also optional features" and "must be explicitly enabled"?
Maybe it would make sense to add this in a subsection (---)?

Reply:
Don't miss the second half of that sentence, "to be included in the build". ie they must be enabled by the user if the user wants their application to have those features. Is it really enough to warrant a separate subsection?

Reply:
A subsection can be even just a couple of words. Here, it would increase visibility of this model paragraph, as it now looks like a clarification about the list that precedes it. And this "also" suggests to me we're talking about something different here.

We should definitely link here to Bluetooth Mesh Models page.
What I'm trying to say is that we need to be more explicit about how to enable models and what it means. The context suggests Kconfig options, but none are present. If "in the build" means a different method, I think it should be expanded or an example should be given. Which will expand this section and warrant a separate subsection even more.

Reply:
The model Kconfig options should be documented in the individual model pages in this case. I can make a separate subsection of this, and rephrase.